core/common/src/main/java/alluxio/underfs/UnderFileSystemConfiguration.java

```diff
@@ -114,6 +118,17 @@ public UnderFileSystemConfiguration setShared(boolean shared) {
   public UnderFileSystemConfiguration createMountSpecificConf(Map<String, String> mountConf) {
     UnderFileSystemConfiguration ufsConf = new UnderFileSystemConfiguration(mProperties.copy());
     ufsConf.mProperties.merge(mountConf, Source.MOUNT_OPTION);
+    String ufsConfigValue = ufsConf.get(PropertyKey.UNDERFS_HDFS_CONFIGURATION);
+    if (ufsConfigValue.startsWith("kv@")) {
+      Configuration hdfsConf = new Configuration();
+      for (String path : ufsConfigValue.substring(3).split(":")) {
```

Comment:
If `"kv@"` was a constant, it can be used for the length.

Reply:
Sure, use length() to get the prefix length.